aiohttp_sse/__init__.py

```diff
@@ -140,8 +140,7 @@ def ping_interval(self):
     def last_event_id(self) -> Optional[str]:
         """Last event ID, requested by client."""
         if self._req is None:
-            msg = "EventSource request must be prepared first"
-            raise RuntimeError(msg)
+            raise RuntimeError("EventSource request must be prepared first")
```

Comment:
I split this one after recommendation: https://docs.astral.sh/ruff/rules/raw-string-in-exception/

Reply:
Hmm, OK.